cmd/deploy.go

```diff
@@ -198,10 +225,13 @@ func runDeploy(cmd *cobra.Command, _ []string, clientFn deployClientFn) (err err
 		return err
 	}
 
-	if config.Build {
+	switch function.BuildType {
```

Comment:
Shouldn't we also handle `BuildTypeDisabled` or `default` case?

Reply:
It shouldn't happen here, but you are right, I will handle that in `default` case. Good catch.